diff -rupN linux-3.15.3-orig/drivers/gpu/drm/gma500/backlight.c linux-3.15.3/drivers/gpu/drm/gma500/backlight.c
--- linux-3.15.3-orig/drivers/gpu/drm/gma500/backlight.c	2014-07-01 04:14:09.000000000 +0100
+++ linux-3.15.3/drivers/gpu/drm/gma500/backlight.c	2014-07-04 12:48:09.749681018 +0100
@@ -26,6 +26,8 @@
 #include "intel_bios.h"
 #include "power.h"
 
+#undef CONFIG_BACKLIGHT_CLASS_DEVICE  // OpenFrame Workaround
+
 #ifdef CONFIG_BACKLIGHT_CLASS_DEVICE
 static void do_gma_backlight_set(struct drm_device *dev)
 {
diff -rupN linux-3.15.3-orig/drivers/gpu/drm/gma500/psb_device.c linux-3.15.3/drivers/gpu/drm/gma500/psb_device.c
--- linux-3.15.3-orig/drivers/gpu/drm/gma500/psb_device.c	2014-07-01 04:14:09.000000000 +0100
+++ linux-3.15.3/drivers/gpu/drm/gma500/psb_device.c	2014-07-04 12:47:59.045680876 +0100
@@ -75,6 +75,8 @@ static int psb_backlight_setup(struct dr
 	uint32_t value;
 	uint32_t blc_pwm_precision_factor;
 
+	return 0;  // OpenFrame Workaround
+
 	/* get bl_max_freq and pol from dev_priv*/
 	if (!dev_priv->lvds_bl) {
 		dev_err(dev->dev, "Has no valid LVDS backlight info\n");
diff -rupN linux-3.15.3-orig/drivers/gpu/drm/gma500/psb_intel_lvds.c linux-3.15.3/drivers/gpu/drm/gma500/psb_intel_lvds.c
--- linux-3.15.3-orig/drivers/gpu/drm/gma500/psb_intel_lvds.c	2014-07-01 04:14:09.000000000 +0100
+++ linux-3.15.3/drivers/gpu/drm/gma500/psb_intel_lvds.c	2014-07-04 12:49:33.069682124 +0100
@@ -172,6 +172,9 @@ static int psb_lvds_pwm_set_brightness(s
  */
 void psb_intel_lvds_set_brightness(struct drm_device *dev, int level)
 {
+
+	dev = "openframe-bl";  // OpenFrame Workaround
+
 	struct drm_psb_private *dev_priv = dev->dev_private;
 
 	dev_dbg(dev->dev, "backlight level is %d\n", level);
