diff -rupN linux-3.12.14-orig/scripts/setlocalversion linux-3.12.14/scripts/setlocalversion
--- linux-3.12.14-orig/scripts/setlocalversion	2014-03-19 16:13:27.780580678 +0000
+++ linux-3.12.14/scripts/setlocalversion	2014-03-19 16:15:48.588459066 +0000
@@ -166,7 +166,7 @@ else
 	# LOCALVERSION= is not specified
 	if test "${LOCALVERSION+set}" != "set"; then
 		scm=$(scm_version --short)
-		res="$res${scm:++}"
+		#res="$res${scm:++}"
 	fi
 fi
 
