--- a/arch/x86/platform/efi/efi_32.c
+++ b/arch/x86/platform/efi/efi_32.c
@@ -54,8 +54,8 @@ void __init efi_map_region(efi_memory_de
 	}

 	md->virt_addr = (unsigned long)va;
-	if (!va)
-		pr_err("ioremap of 0x%llX failed!\n", md->phys_addr);
+	//if (!va)
+	//	pr_err("ioremap of 0x%llX failed!\n", md->phys_addr);
 }

 /*
--- a/drivers/hid/usbhid/hid-core.c
+++ b/drivers/hid/usbhid/hid-core.c
@@ -1357,7 +1357,7 @@ static int usbhid_probe(struct usb_inter
 		if (usb_endpoint_is_int_in(&interface->endpoint[n].desc))
 			has_in++;
 	if (!has_in) {
-		hid_err(intf, "couldn't find an input interrupt endpoint\n");
+		//hid_err(intf, "couldn't find an input interrupt endpoint\n");
 		return -ENODEV;
 	}
