diff -rupN linux-3.14.12-orig/drivers/i2c/busses/i2c-isch.c linux-3.14.12/drivers/i2c/busses/i2c-isch.c
--- linux-3.14.12-orig/drivers/i2c/busses/i2c-isch.c	2014-07-09 19:18:37.000000000 +0100
+++ linux-3.14.12/drivers/i2c/busses/i2c-isch.c	2014-07-16 11:48:50.013444001 +0100
@@ -56,6 +56,8 @@
 #define SCH_WORD_DATA		0x03
 #define SCH_BLOCK_DATA		0x05
 
+#define SMBIOSIZE	8
+
 static unsigned short sch_smba;
 static struct i2c_adapter sch_adapter;
 static int backbone_speed = 33000; /* backbone speed in kHz */
@@ -274,7 +276,7 @@ static int smbus_sch_probe(struct platfo
 	if (!res)
 		return -EBUSY;
 
-	if (!devm_request_region(&dev->dev, res->start, resource_size(res),
+	if (!devm_request_region(&dev->dev, res->start, SMBIOSIZE,
 				 dev->name)) {
 		dev_err(&dev->dev, "SMBus region 0x%x already in use!\n",
 			sch_smba);
