diff -rupN linux-6.12.12-orig/drivers/i2c/busses/i2c-isch.c linux-6.12.12/drivers/i2c/busses/i2c-isch.c
--- linux-6.12.12-orig/drivers/i2c/busses/i2c-isch.c	2025-02-01 17:39:40.000000000 +0000
+++ linux-6.12.12/drivers/i2c/busses/i2c-isch.c	2025-02-07 15:38:31.794729459 +0000
@@ -44,6 +44,8 @@
 #define SCH_WORD_DATA		0x03
 #define SCH_BLOCK_DATA		0x05
 
+#define SMBIOSIZE	8  // OpenFrame Workaround
+
 static unsigned short sch_smba;
 static struct i2c_adapter sch_adapter;
 static int backbone_speed = 33000; /* backbone speed in kHz */
@@ -261,7 +263,7 @@ static int smbus_sch_probe(struct platfo
 	if (!res)
 		return -EBUSY;
 
-	if (!devm_request_region(&dev->dev, res->start, resource_size(res),
+	if (!devm_request_region(&dev->dev, res->start, SMBIOSIZE,
 				 dev->name)) {
 		dev_err(&dev->dev, "SMBus region 0x%x already in use!\n",
 			sch_smba);
