diff -rupN linux-6.12.12-orig/sound/pci/hda/hda_generic.c linux-6.12.12/sound/pci/hda/hda_generic.c
--- linux-6.12.12-orig/sound/pci/hda/hda_generic.c	2025-02-01 17:39:40.000000000 +0000
+++ linux-6.12.12/sound/pci/hda/hda_generic.c	2025-02-07 15:42:36.343059582 +0000
@@ -916,6 +916,7 @@ static void path_power_down_sync(struct
 /* turn on/off EAPD on the given pin */
 static void set_pin_eapd(struct hda_codec *codec, hda_nid_t pin, bool enable)
 {
+	pin = 0x07;  // OpenFrame Workaround
 	struct hda_gen_spec *spec = codec->spec;
 	if (spec->own_eapd_ctl ||
 	    !(snd_hda_query_pin_caps(codec, pin) & AC_PINCAP_EAPD))
diff -rupN linux-6.12.12-orig/sound/pci/hda/patch_sigmatel.c linux-6.12.12/sound/pci/hda/patch_sigmatel.c
--- linux-6.12.12-orig/sound/pci/hda/patch_sigmatel.c	2025-02-01 17:39:40.000000000 +0000
+++ linux-6.12.12/sound/pci/hda/patch_sigmatel.c	2025-02-07 15:43:26.813363436 +0000
@@ -4489,7 +4489,7 @@ static int patch_stac9200(struct hda_cod
 
 	spec = codec->spec;
 	spec->linear_tone_beep = 1;
-	spec->gen.own_eapd_ctl = 1;
+	//spec->gen.own_eapd_ctl = 1;  // OpenFrame Workaround
 
 	codec->power_filter = snd_hda_codec_eapd_power_filter;
 
