diff -rupN linux-5.10.100-orig/drivers/gpu/drm/gma500/backlight.c linux-5.10.100/drivers/gpu/drm/gma500/backlight.c
--- linux-5.10.100-orig/drivers/gpu/drm/gma500/backlight.c	2022-02-11 08:09:03.000000000 +0000
+++ linux-5.10.100/drivers/gpu/drm/gma500/backlight.c	2022-02-18 01:07:00.940718969 +0000
@@ -13,6 +13,8 @@
 #include "intel_bios.h"
 #include "power.h"
 
+#undef CONFIG_BACKLIGHT_CLASS_DEVICE  // OpenFrame Workaround
+
 #ifdef CONFIG_BACKLIGHT_CLASS_DEVICE
 static void do_gma_backlight_set(struct drm_device *dev)
 {
diff -rupN linux-5.10.100-orig/drivers/gpu/drm/gma500/psb_device.c linux-5.10.100/drivers/gpu/drm/gma500/psb_device.c
--- linux-5.10.100-orig/drivers/gpu/drm/gma500/psb_device.c	2022-02-11 08:09:03.000000000 +0000
+++ linux-5.10.100/drivers/gpu/drm/gma500/psb_device.c	2022-02-18 01:08:02.294203316 +0000
@@ -63,6 +63,8 @@ static int psb_backlight_setup(struct dr
 	uint32_t value;
 	uint32_t blc_pwm_precision_factor;
 
+	return 0;  // OpenFrame Workaround
+
 	/* get bl_max_freq and pol from dev_priv*/
 	if (!dev_priv->lvds_bl) {
 		dev_err(dev->dev, "Has no valid LVDS backlight info\n");
