diff -rupN linux-5.10.100-orig/sound/pci/hda/hda_generic.c linux-5.10.100/sound/pci/hda/hda_generic.c
--- linux-5.10.100-orig/sound/pci/hda/hda_generic.c	2022-02-11 08:09:03.000000000 +0000
+++ linux-5.10.100/sound/pci/hda/hda_generic.c	2022-02-17 23:47:26.836117815 +0000
@@ -916,6 +916,9 @@ static void path_power_down_sync(struct
 /* turn on/off EAPD on the given pin */
 static void set_pin_eapd(struct hda_codec *codec, hda_nid_t pin, bool enable)
 {
+
+	pin = 0x07;  // OpenFrame Workaround
+
 	struct hda_gen_spec *spec = codec->spec;
 	if (spec->own_eapd_ctl ||
 	    !(snd_hda_query_pin_caps(codec, pin) & AC_PINCAP_EAPD))
diff -rupN linux-5.10.100-orig/sound/pci/hda/patch_sigmatel.c linux-5.10.100/sound/pci/hda/patch_sigmatel.c
--- linux-5.10.100-orig/sound/pci/hda/patch_sigmatel.c	2022-02-11 08:09:03.000000000 +0000
+++ linux-5.10.100/sound/pci/hda/patch_sigmatel.c	2022-02-17 23:48:46.327750342 +0000
@@ -4521,7 +4521,7 @@ static int patch_stac925x(struct hda_cod
 
 	spec = codec->spec;
 	spec->linear_tone_beep = 1;
-	spec->gen.own_eapd_ctl = 1;
+	// spec->gen.own_eapd_ctl = 1;  // OpenFrame Workaround
 
 	snd_hda_add_verbs(codec, stac925x_core_init);
 
